src/kirby/components/list/list.component.scss

```
@@ -38,4 +36,10 @@ ul {
       }
     }
   }
+}
+
+.clickable {
```

Can be moved up to line 20 under the `li` rule and add a `&`.
```suggestion
&.clickable {
...
}
```

I expect the class `clickable` is going to be used on anything else the `li` tag?